### DISCUSSION TITLE
would it be possible to change the theme using a button in the electron app

### DISCUSSION THREAD

**Discussion by tad-dufort** [Seo-Rii/electron-acrylic-window#94] [Q&A] [answered]
would it be possible to change the theme using a button in the electron app?

I would like to do this but I cant figure out how

**Comment by tscpp** [comment, marked as answer]
I transferred your issue to a discussion. Questions do not belong as an issue. You can also ask questions on a forum like stackoverflow.

Here are some ways of communicating with the main process from the renderer:
1. https://github.com/electron/remote
2. https://www.electronjs.org/docs/latest/tutorial/tutorial-preload
3. https://www.electronjs.org/docs/latest/api/context-bridge
